### ISSUE TITLE
Is it necessary to enable choosing different encoding in `parse_file()`?

### ISSUE THREAD

**Issue by himcraft** [sciunto-org/python-bibtexparser#394]
When I load my `.bib` file, it prompts that 
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "D:\python310\lib\site-packages\bibtexparser\entrypoint.py", line 114, in parse_file
    bibtex_str = f.read()
UnicodeDecodeError: 'gbk' codec can't decode byte 0x99 in position 957: illegal multibyte sequence
```

The problem can be solved by starting python by `python -Xutf8`. But I want to know if you can add the option to choose a different encoding scheme? Like `bibtexparser.parse_file("test.bib",encoding="UTF-8")`?

Thanks!

**Comment by himcraft** [comment]
Thank you! I will try to submit a PR later :)

**Comment by MiWeiss** [comment]
Closed in #395
